java/code/src/com/suse/oval/OVALCachingFactory.java

```diff
@@ -53,6 +53,8 @@ private OVALCachingFactory() {
      *
      * @param rootType the OVAL root to extract from
      * */
+    // TODO: Following the single responsibility principle this method shouldn't have to extract vulnerable packages
```

Comment:
If you don't plan to address this in this PR please add a card for it. We should try to avoid merging the code with leftover todos.

Reply:
Ah okay, I use TODOs to not forget about things, but you're right I don't plan to work on this in the current PR.